source/api/content-state/0.3/index.md

```diff
@@ -226,6 +226,9 @@ The destination character set when encoding _MUST_ be UTF-8. For example, when p
 
 When published as inline, base64url-encoded JSON-LD in the full form given in 2.2, the content state Annotation _MAY_ omit the `id` and `@context` properties.
 
+When published on a server for clients to fetch over HTTP, content states _MUST_ be valid JSON-LD documents conforming to the [IIIF Presentation API][prezi-api] and served as described in [Section 5][contentstate-http] below, and _MUST NOT_ be encoded as base64url or any other encoding.
```

Comment:
The "MOST NOT be encoded" seems strong. Would it be OK to provide a response with `Content-Encoding: gzip` ? I would remove the "or any other encoding" to admit the possibility of existing mechanisms (which don't include b64).

Reply:
addressed in imminent PR